docs/migration.md

```diff
@@ -4,19 +4,38 @@
 
 This release removes the old, HTTP method-based scopes and replaces them with
 the new, functional-motivated scopes `ADMIN` and `ME`. The auth-server will
-automatically migrate all clients to the new scopes in the following way:
+automatically migrate all well-known clients to the new scopes in the following
+way:
 
 1. Remove the old scopes (`GET`, `POST`, `PUT`, `PATCH`, `DELETE`)
 2. Add the new scopes (`ADMIN`, `ME`)
 
-It will not touch self-defined scopes! Please check the scopes of your clients
-after the migration to ensure that everything is like you would expect. You
-maybe have to remove the new scopes from your clients if they don't use them.
+Well-known clients and their scopes are:
+
+* `example-client` (`ADMIN`, `ME`)
+* `auth-server` (`ADMIN`)
+* `addon-self-administration-client` (`ADMIN`)
+* `addon-administration-client` (`ADMIN`)
+
+The migration will not touch self-defined scopes nor self-defined clients!
+Before deploying the auth-server and resource-server (and thus run the automatic
+migration) update your existing clients:
+
+1. Examine the current scopes and usages of the client.
+2. Should access tokens granted to this client have full administrative access to OSIAM?
+  1. Add scope `ADMIN`
+3. Should access tokens granted to this client have only access to the associated user?
+  1. Add scope `ME`
+4. Remove the old scopes (`GET`, `POST`, `PUT`, `PATCH`, `DELETE`)
+5. Deploy the auth-server
+
+**Note:** The scopes `ADMIN` and `ME` are supported since the last release, so
```

Comment:
This sentence reads a little clunky in my opinion. Let me suggest this alternative wording: "The new scopes `ADMIN` and  `ME` have been supported since the last release, to make it possible to just switch to them without upgrading OSIAM and still have everything work as expected. This way you can schedule an upgrade at your own convenience and minimize downtime."